docs/management_api_v2.yml

```diff
@@ -44,6 +44,182 @@ responses:
       $ref: "#/definitions/Error"
 
 paths:
+  /deployments/releases:
+    get:
+      operationId: List Releases with pagination
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: |
+        List releases
+      description: |
+        Returns a collection of releases, allows filtering by release name and sorting
+        by name or last modification date.
+      parameters:
+        - name: name
+          in: query
+          description: Release name filter.
+          required: false
+          type: string
+        - name: device_type
```

Comment:
do we still want to filter by update type? is it included in the new UX for releases?

Reply:
update type yes ([see here](https://www.figma.com/file/xHtArTgENL7BZHGBNYX1Vr/Delta-artifacts?node-id=809%3A1296&mode=dev))
OTOH `device_type` is not in the UX, but not sure if this is needed elsewhere...

Reply:
right, sorry; I meant `device_type`

Reply:
to me looks kind of "obvious" to provide a filter by device_type, I'd keep it at least in the API.
The cost of this extra filter is minimal. Do you disagree, @kjaskiewiczz (and @alfrunes, @merlin-northern, @mzedel)?

Reply:
EDIT: changed my mind, let's drop it.